mintlify/mint
# Refurbish cli

## Summary
This PR allows the current CLI architecture to run off of our main branch.

## To-Do
- [x] Merge #151 
^ Need to merge to get the different ports working
- [x] Create a PR that turns multi-tenancy off by default
^ Not necessary anymore
- [x] Get listener to work
  - [x] Make `CMD_EXEC_PATH` and `CLIENT_PATH` inputs in mintlify-file-listening
  - [x] Properly publish mintlify-file-listening. Previously we were not uploading it to the registry. I uploaded it but it isn't configured properly. Inside my node_modules, you can see the `bin` folder is not being exported.
![image](https://user-images.githubusercontent.com/55263191/212028270-801bacfc-2fc5-49b9-b487-e70d5a37b573.png)
^ abandoning `mintlify-file-listening` just copied over the files.


## Test Plan
1. Run `npm i -g .`. For some reason `yarn link` was not working for me so I had to resort to using the old command.
2. Navigate to a local docs instance
3. Run `mintlify dev`
![image](https://user-images.githubusercontent.com/55263191/212029654-ce4822f5-7225-45e5-8bc1-687e1d25efbe.png)

- Listener should be working now.
